docs/design/adminconsole/TENANT-DATA.md

```diff
@@ -0,0 +1,37 @@
+# Tenant Management Data
+
+See [APIs for Admin Console: Tenant
+Management](./APIS-FOR-ADMIN-CONSOLE.md#tenant-management) for context.
+
+> [!NOTE]
+> Full CRUD support with database backend will be covered in a future version.
+
+## REST Interface
+
+### GET /adminconsole/tenants
+
+Also supports `GET /adminconsole/tenants/{id}`
+
+* **Purpose**: Provide tenant list to Admin Console.
+* **Description**:
+  * Reads from the appsettings file.
+  * No additional authorization required.
+  * Respond with 200
+* **Response Format**:
+
+  ```json
+  [
+    {
+      "tenantId": 1,
+      "document": {
+        "name": "Tenant 1",
```

Comment:
I know this is just an example, but might help somebody. Tenant names cannot have white spaces.  And they won't because this is read from the appsettings -> `Tenants` section. [Multi-Tenant Configuration](https://docs.ed-fi.org/reference/admin-api/admin-api-2.x/technical-articles/multi-tenant-configuration-for-admin-api-2x)

Reply:
Good to know. I will update accordingly. Just curious, why are spaces not allowed?

Reply:
Glad you pointed me to this document. I've been meaning to look for this document. I see that we need to expand the appsettings file to include the discoveryApiUrl.

Reply:
Could have been a wrong assumption from my side. Since these values are part of the `resourceUrl` and `TokenUrl` (Necessary to access Ods Api) I am assuming that tenant name should not have spaces.

Reply:
You may be correct. Or, the space might be URL encoded with a `+`.